src/Template/ModelTemplate.cs

```diff
@@ -257,7 +273,12 @@ public void Write(ClientModel model, JavaFile javaFile)
                                 }
                                 else
                                 {
-                                    methodBlock.Line($"this.{property.Name} = {expression};");
+                                    if (property.UseVarArg) {
```

Comment:
curious if the conditional operator works in java?

`methodBlock.Line($"this.{property.Name} = property.UseVarArg ? Arrays.asList({expression}) : {expression};");`